tests/network/network_service/conftest.py

```
@@ -13,6 +15,31 @@
+    expose_command = basic_expose_command(resource_name=running_vm_for_exposure.name, svc_name=service_name)
+    expose_command += f" --ip-family-policy={ip_family_policy}"
+    run_virtctl_command(command=shlex.split(expose_command), namespace=running_vm_for_exposure.namespace, check=True)
+
+    service = Service(
+        name=service_name,
+        namespace=running_vm_for_exposure.namespace,
+        client=client,
+    )
+    for sample in TimeoutSampler(wait_timeout=15, sleep=1, func=lambda: service.exists):
+        if sample:
+            LOGGER.info(f"Service {service_name} was successfully created")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)